### PR TITLE
feat: huyi sms

### PR DESCRIPTION
**Pull request by Curlson**
feat huyiwuxian sms provider

### CODE REVIEW

File: huyi.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	v.Set("account", hc.appId)
+	v.Set("content", smsContent)
+	v.Set("time", _now)
+	passwordStr := hc.appId + hc.appKey + "%s" + smsContent + _now
+	for _, mobile := range targetPhoneNumber {
+		password := fmt.Sprintf(passwordStr, mobile)
+		v.Set("password", GetMd5String(password))
+		v.Set("mobile", mobile)
+
+		body := strings.NewReader(v.Encode()) //把form数据编下码
```
Comment (hsluoyz) [inline huyi.go]: Plz use English comments

File: huyi.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
+package go_sms_sender
```
Comment (hsluoyz) [inline huyi.go:15]: Add license header

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by Curlson** [comment]
add license header and translate comments to eng

**Comment by hsluoyz** [comment]
lgtm

**Pull request merged**